apps/rig_api/lib/rig_api/controllers/message_controller.ex

```diff
@@ -60,6 +60,14 @@ defmodule RigApi.MessageController do
           description("The broadcasted CloudEvent according to the CloudEvents spec.")
 
           properties do
+            id(
+              :string,
+              "ID of the event. The semantics of this string are explicitly undefined to ease \
+              the implementation of producers. Enables deduplication.",
+              required: true,
+              example: "A database commit ID"
+            )
+
```

Comment:
isn't `source` required too?

Reply:
Should be, but apparently not in our code. At least I didn't see any error when using `/messages` endpoint.

Reply:
created an issue for this: #191